.travis.yml

```diff
@@ -23,6 +30,8 @@ install:
   - popd
 
 before_script:
+  - sudo sed -i 's/port = 5433/port = 5432/' /etc/postgresql/11/main/postgresql.conf
```

Comment:
- [ ] You can access port 5433 by changing the port accessed from the data source.
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/8216fd99ebac06b03a43009690f328071326834c/terasoluna-gfw-functionaltest-env/configs/tomcat9-postgresql/ContainerConfigXML/context.xml#L8

And also `sql-maven-plugin` settings.
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/8216fd99ebac06b03a43009690f328071326834c/terasoluna-gfw-functionaltest-initdb/pom.xml#L27

And also `PGPORT` env var for `createdb`.

Reply:
The application absorbs the effects of the port change.
There are two places affected by the port change
- https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/master/terasoluna-gfw-functionaltest-initdb/pom.xml#L27
- https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/master/terasoluna-gfw-functionaltest-env/configs/travis/ContainerConfigXML/context.xml#L8

When you add test data to the DB by using initdb, specify the value of the url by `-Ddb.url`.
The context.xml was changed from port 5432 to port 5433 using the direct sed command.
I considered changing it with properties, but I can't find a way to reference it from context.xml, so I'll use the sed command.

Also, I changed the configuration to allow connections from localhost, because the default peer authentication settings will play the connection.
37a74f3ef7f5db0c269ccc9c88fca310fe59193b

Reply:
Fix to allow reference to a value using a placeholder.
Because can't specify a default value like `${property:default value}`, placeholder is defined with the property `cargo.connect.db.port`.
07278b610b8508bb424d350e403ebb0816ae9d65

Reply:
I think you only have to specify the port number.
```
-Dcargo.connect.db.port=5433
(omitted)
<cargo.connect.db.port>5432</cargo.connect.db.port>
<cargo.jvmargs>-Ddb.port=${cargo.connect.db.port} -Xms512m...
```

Reply:
The policy was to use the sed command to set it up, so we revert unneeded commits.
4897d819d92734666fa143f113bb00c29f8c8019
dcd7e1bf581a99e4e191fdc6d6e17ab7046793d8